Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/clerk.yml

```diff
@@ -11,4 +11,5 @@
     - id: BoxEncryptionKeyJustice
     - id: ClerkIDCard
     - id: RubberStampNotary
+    - id: LunchboxJusticeFilledRandom # Delta-V Lunchboxes!
```

Comment:
dont need deltav comment in deltav files

Reply:
pro outdated now